AlgebraicPetri-Stratification/src/GrometInterop.jl

```diff
@@ -161,7 +161,7 @@ function semagram2petrinet(sg)
           pn[j_ind, :sname] = Symbol(j["name"])
         end
         if valued
-          pn[j_ind, :concentration] = isnothing(j["value"]) ? 0.0 : j["value"]
+          pn[j_ind, :concentration] = isnothing(j["value"]) ? 0.0 : parse(Float64, j["value"]["value"]["val"])
```

Comment:
Could this be `parse(c_type, j["value"]["value"]["val"])`? This could ensure that the concentration value data type is consistent with the data type specified in the Gromet.

Reply:
Fixed. Thanks!